test/linalg/dense.jl

```diff
@@ -185,354 +194,372 @@ for elty in (Int32, Int64, Float32, Float64, Complex64, Complex128)
+                    @test norm(α*A,Inf) ≈ abs(α)*norm(A,Inf)
+
+                    # Triangle inequality
+                    @test norm(A + B,1) <= norm(A,1) + norm(B,1)
+                    elty <: Union{BigFloat,Complex{BigFloat},BigInt} || @test norm(A + B) <= norm(A) + norm(B) # two is default
+                    @test norm(A + B,Inf) <= norm(A,Inf) + norm(B,Inf)
+
+                    # vecnorm:
+                    for p = -2:3
+                        @test norm(reshape(A, length(A)), p) == vecnorm(A, p)
+                    end
+                end
             end
-        end
 
-        # issue #10234
-        if elty <: AbstractFloat || elty <: Complex
-            let z = zeros(elty, 100)
-                z[1] = -Inf
-                for p in [-2,-1.5,-1,-0.5,0.5,1,1.5,2,Inf]
-                    @test norm(z, p) == (p < 0 ? 0 : Inf)
-                    @test norm(elty[Inf],p) == Inf
+            @testset "issue #10234" begin
+                if elty <: AbstractFloat || elty <: Complex
+                    let z = zeros(elty, 100)
+                        z[1] = -Inf
+                        for p in [-2,-1.5,-1,-0.5,0.5,1,1.5,2,Inf]
+                            @test norm(z, p) == (p < 0 ? 0 : Inf)
+                            @test norm(elty[Inf],p) == Inf
+                        end
+                    end
                 end
             end
         end
     end
-end
 
-# issue #10234
-@test norm(Any[Inf],-2) == norm(Any[Inf],-1) == norm(Any[Inf],1) == norm(Any[Inf],1.5) == norm(Any[Inf],2) == norm(Any[Inf],Inf) == Inf
+    @testset "issue #10234" begin
+        @test norm(Any[Inf],-2) == norm(Any[Inf],-1) == norm(Any[Inf],1) == norm(Any[Inf],1.5) == norm(Any[Inf],2) == norm(Any[Inf],Inf) == Inf
+    end
 
-# overflow/underflow in norms:
-@test norm(Float64[1e-300, 1], -3)*1e300 ≈ 1
-@test norm(Float64[1e300, 1], 3)*1e-300 ≈ 1
+    @testset "overflow/underflow in norms" begin
+        @test norm(Float64[1e-300, 1], -3)*1e300 ≈ 1
+        @test norm(Float64[1e300, 1], 3)*1e-300 ≈ 1
+    end
+end
 
 ## Issue related tests
-# issue #1447
-let
-    A = [1.+0.0im 0; 0 1]
-    B = pinv(A)
-    for i = 1:4
-        @test A[i] ≈ B[i]
+@testset "issue #1447" begin
+    let
+        A = [1.+0.0im 0; 0 1]
+        B = pinv(A)
+        for i = 1:4
+            @test A[i] ≈ B[i]
+        end
     end
 end
 
-# issue #2246
-let
-    A = [1 2 0 0; 0 1 0 0; 0 0 0 0; 0 0 0 0]
-    Asq = sqrtm(A)
-    @test Asq*Asq ≈ A
-    A2 = view(A, 1:2, 1:2)
-    A2sq = sqrtm(A2)
-    @test A2sq*A2sq ≈ A2
+@testset "issue #2246" begin
+    let
+        A = [1 2 0 0; 0 1 0 0; 0 0 0 0; 0 0 0 0]
+        Asq = sqrtm(A)
+        @test Asq*Asq ≈ A
+        A2 = view(A, 1:2, 1:2)
+        A2sq = sqrtm(A2)
+        @test A2sq*A2sq ≈ A2
+    end
+
+    let
+        N = 3
+        @test log(det(eye(N))) ≈ logdet(eye(N))
+    end
 end
 
-let
-    N = 3
-    @test log(det(eye(N))) ≈ logdet(eye(N))
+@testset "issue #2637" begin
+    let
+      a = [1, 2, 3]
```

Comment:
This block wasn't indented to the typical multiple of 4 spaces. The `let` is also redundant with the testset now.